event-details-boundary/attendees-list/EventAttendeesList.test.ts

```diff
@@ -0,0 +1,82 @@
+import { EventAttendeeMocks } from "@event-details-boundary/MockData"
+import { uuidString } from "@lib/utils/UUID"
+import { EventAttendeesList } from "./EventAttendeesList"
+
+describe("EventAttendeesList tests", () => {
+  describe("EventAttendeesListClass tests", () => {
+    const mockAttendeeData = {
+      attendees: [EventAttendeeMocks.Alivs, EventAttendeeMocks.AnnaAttendee],
+      totalAttendeeCount: 2,
+      nextPageCursor: null
+    }
+    const mockAttendeeData2 = {
+      attendees: [EventAttendeeMocks.BlobJr, EventAttendeeMocks.BlobSr],
+      totalAttendeeCount: 4,
+      nextPageCursor: null
+    }
+    const testAttendeesList = new EventAttendeesList([
+      mockAttendeeData,
+      mockAttendeeData2
+    ])
+    test("total attendee count", () => {
+      expect(testAttendeesList.totalAttendeeCount).toEqual(4)
+    })
+    test("host", () => {
+      expect(testAttendeesList.host).toEqual(EventAttendeeMocks.Alivs)
+    })
+    test("attendees", () => {
+      expect(testAttendeesList.attendees()).toEqual([
+        EventAttendeeMocks.AnnaAttendee,
+        EventAttendeeMocks.BlobJr,
+        EventAttendeeMocks.BlobSr
+      ])
+    })
+    it("switches host with other attendee in list", () => {
+      const attendeesList = testAttendeesList.swapHost(
+        EventAttendeeMocks.BlobJr.id
+      )
+      expect(attendeesList.host).toEqual(EventAttendeeMocks.BlobJr)
+      expect(attendeesList.attendees()).toEqual([
+        EventAttendeeMocks.AnnaAttendee,
+        EventAttendeeMocks.Alivs,
+        EventAttendeeMocks.BlobSr
+      ])
+    })
+    it("switches host with attendee right after original host in list", () => {
```

Comment:
Shouldn't these test names mention that page of the attendee we're swapping the host with? 1 test seems like it was meant for swapping the host with an attendee on the first page, and the other the second page. "list" and "original list" was a bit confusing, so I think "first page" and "second page" would work better.